app/grandchallenge/algorithms/views.py

```diff
@@ -85,7 +85,7 @@ def form_valid(self, form):
         return response
 
 
-class AlgorithmList(PermissionListMixin, FilterMixin, ListView):
+class AlgorithmList(FilterMixin, PermissionListMixin, ListView):
```

Comment:
Why have the mixins changed order here? Is this to get the number of objects pre-permissions filtering?

Reply:
The other way around. It's to get the correct number of objects on the context generation in the `FilterMixin`; post-permissions. The method resolution order had the permissions exclude objects after the `FilterMixin` making the mixin count objects that would not have been visible to the user anyway.

Reply:
You're right, my mistake!